### DISCUSSION TITLE
Clarification on task name length limit

### DISCUSSION THREAD

**Discussion by HalavicH** [kagkarlsson/db-scheduler#412] [Q&A] [answered]
Hello Karlsson and the community. I want to express my gratitude for developing such a good library and leading this community.

I have observed that there is a restriction on the length of a task, limiting it to 100 characters. Is it still applicable?
I'm curious about the rationale behind this limitation. Is it feasible to extend the task length to 255 characters with some tweaks/tricks?
I'm particularly interested in understanding how this limitation applies to Oracle DB.

Thanks in advance

**Comment by kagkarlsson** [comment, marked as answer]
I don't think there is a truncation of the task-name length, just accidental limit of 100chars in the proposed schema for Oracle. Postgres schema does not have a limit. So I think you can change it

**Reply by HalavicH** [reply]
Thanks for the clarification!
